dives/DL7.zxu

```diff
@@ -9,6 +9,6 @@ ZDP{
 |3300|10|||||
 |3600|0|||||
 ZDP}
-ZDT|2|2|10.0|20180102110000|25||
-ZDH|3|3|I|QS|20180103101000|28|11|FO2|||
-ZDT|3|3|10.0|20180103102000|26||
+ZDT|1|2|10.0|20180102110000|25||
```

Comment:
This seems to be leaving this record with an inconsistent export id between head and tail - probably not helpful to have this in a test file.